confluent_kafka/avro/__init__.py

```diff
@@ -0,0 +1,16 @@
+VALID_LEVELS = ['NONE', 'FULL', 'FORWARD', 'BACKWARD']
+
+
+class ClientError(Exception, object):
+    """Error thrown by Schema Registry clients"""
+
+    def __init__(self, message, http_code=-1):
```

Comment:
http_code is integer number in all the places. So -1 was considered. I shall change it to None.